bundles/org.openhab.binding.mffan/src/main/java/org/openhab/binding/mffan/internal/handler/ShadowBufferDto.java

```diff
@@ -0,0 +1,223 @@
+/**
```

Comment:
I would suggest to move the API to a different folder and package. 
bundles/org.openhab.binding.mffan/src/main/java/org/openhab/binding/mffan/internal/api/
Files `FanRestApi.java`, `RestApiException.java` and `ShadowBufferDto.java`